packages/manager/modules/netapp/src/index.scss

```diff
@@ -0,0 +1,7 @@
+.netapp-section {
```

Comment:
This could have just been applied to netapp component, no need to create a class :)